airbyte-integrations/bases/standard-destination-test/src/main/resources/v1/data_type_basic_test_messages.txt

```diff
@@ -5,21 +5,21 @@
 {"type": "RECORD", "record": {"stream": "date_test_1", "emitted_at": 1602637589200, "data": { "data" : "1504-02-29" }}}
 {"type": "RECORD", "record": {"stream": "date_test_1", "emitted_at": 1602637589300, "data": { "data" : "9999-12-23" }}}
 {"type": "RECORD", "record": {"stream": "datetime_test_1", "emitted_at": 1602637589100, "data": { "data" : "2022-01-23T01:23:45Z" }}}
-{"type": "RECORD", "record": {"stream": "datetime_test_1", "emitted_at": 1602637589200, "data": { "data" : "2022-01-23T01:23:45.678-11:30 BC" }}}
+{"type": "RECORD", "record": {"stream": "datetime_test_1", "emitted_at": 1602637589200, "data": { "data" : "2022-01-23T01:23:45-11:30" }}}
 {"type": "RECORD", "record": {"stream": "datetime_test_1", "emitted_at": 1602637589300, "data": { "data" : "9999-12-23T01:23:45Z" }}}
 {"type": "RECORD", "record": {"stream": "datetime_test_2", "emitted_at": 1602637589100, "data": { "data" : "2022-11-22T01:23:45" }}}
 {"type": "RECORD", "record": {"stream": "datetime_test_2", "emitted_at": 1602637589200, "data": { "data" : "1504-02-29T01:23:45" }}}
 {"type": "RECORD", "record": {"stream": "datetime_test_2", "emitted_at": 1602637589300, "data": { "data" : "9999-12-21T01:23:45" }}}
 {"type": "RECORD", "record": {"stream": "time_test_1", "emitted_at": 1602637589100, "data": { "data" : "01:23:45Z" }}}
-{"type": "RECORD", "record": {"stream": "time_test_1", "emitted_at": 1602637589200, "data": { "data" : "01:23:45.678-11:30" }}}
+{"type": "RECORD", "record": {"stream": "time_test_1", "emitted_at": 1602637589200, "data": { "data" : "01:23:45-11:30" }}}
 {"type": "RECORD", "record": {"stream": "time_test_2", "emitted_at": 1602637589300, "data": { "data" : "01:23:45" }}}
-{"type": "RECORD", "record": {"stream": "time_test_2", "emitted_at": 1602637589100, "data": { "data" : "01:23:45.678" }}}
-{"type": "RECORD", "record": {"stream": "number_test_1", "emitted_at": 1602637589200, "data": { "data" : "56.78" }}}
```

Comment:
what's the purpose of changing these emitted_at timestamps?

Reply:
In some of DBs we relay on emittedAt inserting order.  So if EmittedAt does not correspond to the order in test file - then the test will compare the wrong values.